src/PlaywrightSharp.Tests/BrowserTypeLaunchTests.cs

```diff
@@ -171,5 +171,20 @@ public async Task ShouldWorkWithEnvironmentVariables()
 
             await using var browser = await BrowserType.LaunchAsync(options);
         }
+
+        /// <summary>
+        /// PuppeteerSharp test. It's not in upstream
```

Comment:
Will it be? Should it be?

Reply:
We have this method https://github.com/microsoft/playwright-sharp/blob/afb06a5a3c69f48f524260ad382aab59efd13e77/src/PlaywrightSharp/BrowserType.cs#L268
Which internally converts a `LaunchOption` to a `LaunchPersistentOption`.
So I'm validating that if you want to use a `LaunchOption` you didn't set the `FirefoxUserPrefs`. Typescript doesn't have this feature. wdyt?